docs/web-ui.md

```diff
@@ -369,3 +369,44 @@ The web UI includes a Streaming tab if the application uses Spark streaming. Thi
 scheduling delay and processing time for each micro-batch in the data stream, which can be useful
 for troubleshooting the streaming application.
 
+## JDBC/ODBC Server Tab
+We can see this tab when Spark is running as a [distributed SQL engine](sql-distributed-sql-engine.html) and shows information about sessions and SQL jobs submitted.
+
+At the beginning of the page is the server start time.
+
+<p style="text-align: center;">
+  <img src="img/JDBCServer1.png" width="40%" title="JDBC/ODBC Header" alt="JDBC/ODBC Header">
+</p>
+
+The first section contains information of active and finished sessions with some information.
```

Comment:
The second section contains information about active and finished sessions.
(`first` -> `second`; `information of` -> `information about`; skip `with some information`)